install-template.sh

```diff
@@ -24,6 +24,34 @@ install_nix () {
   set -eu
 }
 
+test_cache_configured () {
+  nixConfig=$(nix --extra-experimental-features 'nix-command flakes' show-config) &&
+  (echo "$nixConfig" | grep --quiet "^substituters.*cache\.garnix\.io") &&
```

Comment:
This won’t match “extra-substituers”, just “substitutes”, no? Shouldn’t the ^ be removed?

Reply:
No, `nix show-config` doesn't show the `extra-{substituters,trusted-public-keys}` fields, but only the aggregated results. So in there it actually says `^substituters: <list of all substituters, regardless of how they're added>`.